pkg/kubelet/checkpoint/checkpoint.go

```diff
@@ -39,54 +36,57 @@ const (
 	podPrefix = "Pod"
 )
 
-// Manager is the interface used to manage checkpoints
-// which involves writing resources to disk to recover
-// during restart or failure scenarios.
-// https://github.com/kubernetes/community/pull/1241/files
-type Manager interface {
-	// LoadPods will load checkpointed Pods from disk
-	LoadPods() ([]*v1.Pod, error)
-
-	// WritePod will serialize a Pod to disk
-	WritePod(pod *v1.Pod) error
+// Data to be stored as checkpoint
+type Data struct {
+	Pod      *v1.Pod
+	CheckSum uint64
```

Comment:
Checksum is camel-case for this struct but the functions are cased as UpdateChecksum. Can we change this field to just Checksum?